src/TraceEvent/BPerf/BPerfEventSource.cs

```diff
@@ -610,6 +631,19 @@ private static long GetOffset(string indexFile, DateTime requestTimestamp, out l
             return 0;
         }
 
+        /// <summary>
+        /// The delegate function to decompress by using ULZ777 algorithm
+        /// </summary>
+        /// <returns></returns>
+        public unsafe static int ULZ777Decompress(byte[] input, int inputOffset, int inputLength, byte[] output, int outputLength)
```

Comment:
Do you need this to be public?  I'd recommend marking this private and just let it be an implementation detail.

Reply:
changed to private, thanks